test/functional.spec.js

```diff
@@ -116,7 +136,7 @@ describe('Functional tests', () => {
     });
 
     it('should call DES encrypt if encoding is DES', () => {
-      lib = new SecureLS({encodingType: 'DES'});
+      lib = new SecureLS({encodingType: 'DES', encryptionSecret: 'some', encryptionNamespace: 'some'});
```

Comment:
@softvar I did it to prevent the test from writing a lot of warnings. There is no default password anymore. Thats why a user should be warned that the encryption is insecure, but not in the test case.